config/exercise_readme.go.tmpl

```diff
@@ -18,6 +18,19 @@ To raise a message with an exception, just write it as an argument to the except
 raise Exception("Meaningful message indicating the source of the error")
 ```
 
+## Running the tests
+
+To run the tests run the command `python -m pytest {{ .Spec.SnakeCaseName }}_test.py`
+(`py.test {{ .Spec.SnakeCaseName }}_test.py` for Python 2; `pytest <exercise>_test.py` for Python 3). Or run it with
+`-v` to be more verbose like this: `pytest {{ .Spec.SnakeCaseName }}_test.py`. Using the
+`-x` flag will cause the test to fail at the first failure instead of it
+trying all cases before exiting.
+
+To run this exercises's tests for example, run this command:
+```
+python -m pytest {{ .Spec.SnakeCaseName }}_test.py -v
+```
```

Comment:
I find this section a little bit hard to scan. How about something like:

------

The command to run the tests is:

- (Python 2) `python -m py.test {{ .Spec.SnakeCaseName }}_test.py`
- (Python 3) `python -m pytest {{ .Spec.SnakeCaseName }}_test.py`

Additional flags are available for pytest:

- `-v` to get more verbose failure messages
- `-x` to fail at the first failure instead of running all the tests and showing all the failures at once

For example:

```
python -m pytest -x {{ .Spec.SnakeCaseName }}_test.py
```

Reply:
I've rewritten this section for clarity; however, it just occurred to me that while tables are valid markdown, they might be discouraged in this instance to make it easier for users to view the README file in a text editor. If that is the case, I am happy to convert my changes to a similar format to what you have written.

Reply:
Yeah, this needs to be readable in plain text.